js/src/interface.js

```diff
@@ -283,12 +324,8 @@ let app = new Vue({
 
 // Fired when the external save button is clicked
 Fliplet.Widget.onSaveRequest(() => {
-
   // Send back the result
-  Fliplet.Widget.save(JSON.parse(JSON.stringify({
-    settings: settings,
-    result: app.result
-  }))).then(() => {
+  Fliplet.Widget.save(app.result).then(() => {
```

Comment:
@tonytlwu well then feel free to only return the results, but I still suggest keeping the JSON trick (and perhaps add a comment for the next guy)